apiserver/README.md

```diff
@@ -119,744 +153,1941 @@ nodes:
+            }
+          },
+          {
+            "deploymentName": "FruitMarket",
+            "replicas": 1,
+            "actorOptions": {
+              "cpusPerActor": 0.1
+            }
+          },
+          {
+            "deploymentName": "DAGDriver",
+            "replicas": 1,
+            "routePrefix": "/",
+            "actorOptions": {
+              "cpusPerActor": 0.1
+            }
+          }
+        ]
       },
-      {
-        "deploymentName": "FruitMarket",
-        "replicas": 1,
-        "actorOptions": {
-          "cpusPerActor": 0.1
-        }
-      },{
-        "deploymentName": "DAGDriver",
-        "replicas": 1,
-        "routePrefix": "/",
-        "actorOptions": {
-          "cpusPerActor": 0.1
-        }
-      }]
-  },
-  "clusterSpec": {
-    "headGroupSpec": {
-      "computeTemplate": "default-template",
-      "image": "rayproject/ray:2.6.3",
-      "serviceType": "NodePort",
-      "rayStartParams": {
+      "clusterSpec": {
+        "headGroupSpec": {
+          "computeTemplate": "default-template",
+          "image": "rayproject/ray:2.6.3-py310",
+          "serviceType": "NodePort",
+          "rayStartParams": {
             "dashboard-host": "0.0.0.0",
             "metrics-export-port": "8080"
+          },
+          "volumes": []
         },
-       "volumes": []
-    },
-    "workerGroupSpec": [
-      {
-        "groupName": "small-wg",
-        "computeTemplate": "default-template",
-        "image": "rayproject/ray:2.6.3",
-        "replicas": 1,
-        "minReplicas": 0,
-        "maxReplicas": 5,
-        "rayStartParams": {
-                "node-ip-address": "$MY_POD_IP"
+        "workerGroupSpec": [
+          {
+            "groupName": "small-wg",
+            "computeTemplate": "default-template",
+            "image": "rayproject/ray:2.6.3-py310",
+            "replicas": 1,
+            "minReplicas": 0,
+            "maxReplicas": 5,
+            "rayStartParams": {
+              "node-ip-address": "$MY_POD_IP"
             }
+          }
+        ]
       }
-    ]
-  }
-}'
-```
-The Ray resource will then be created in your Kubernetes cluster.
+    }'
+
+    # Get the pods running in the namespace
+    kubectl get pods -n ray-system
+    # NAME                                             READY   STATUS    RESTARTS   AGE
+    # test-v1-raycluster-7c2n7-head-5qpwj              1/1     Running   0          14m
+    # test-v1-raycluster-7c2n7-worker-small-wg-9dbdk   1/1     Running   0          14m
+
+    # Delete the RayService in the namespace
+    curl --silent -X 'DELETE' \
+    'http://localhost:31888/apis/v1alpha2/namespaces/ray-system/services/test-v1' \
+    -H 'accept: application/json'
+    ```
+
+## Swagger Support
 
-## Full definition of payload
+Kuberay API server has support for Swagger UI and can be reached at [localhost:31888/swagger-ui](localhost:31888/swagger-ui) for local deployments or for nodeport deployments `<host name>:31888/swagger-ui`
```

Comment:
```suggestion
Kuberay API server has support for Swagger UI and can be reached at [localhost:8888/swagger-ui](localhost:8888/swagger-ui) for local deployments or for nodeport deployments `<host name>:31888/swagger-ui`
```

Reply:
I will reword this a little, but this guide is mostly geared toward the kind (k8s) deployments. Port 8888 would not work for a kind deployment.

Reply:
I see. I was thinking local development with `make run`.